fe/fe-core/src/main/java/org/apache/doris/nereids/util/Utils.java

```diff
@@ -39,8 +39,49 @@ public static String quoteIfNeeded(String part) {
     }
 
     /**
-     * Fully qualified identifier name parts, i.e., concat qualifier and name into a list.
```

Comment:
Please append new functions at the bottom of the file.

Reply:
I would note this next time, we didn't do as that is because I thought place related funciton together may be easy to read.

Reply:
You deleted the comments of `qualifiedNameParts` by mistake...